### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by DLWoodruff**
rearrange it a  little bit
reference the MPC paper instead of OOL

### CODE REVIEW

**Comment by DLWoodruff** [comment]
I already merged a change that should fix the pyotracker failure

File: README.rst [hunk 1]
```diff
@@ -4,18 +4,7 @@ mpi-sppy
...
-^^^^^^
-
-There was a disruptive change on August 11, 2022 concerning how
-options are accessed. See the file ``disruptions.txt`` for more
-information. If you are a new user, this will not affect you,
-regardless of how you install. If you are an
-existing user, you should consider the disruption before updating to
-the latest mpi-sppy. The documentation on readthedocs
-probably refers to the newest version.
+a there is a `paper <https://rdcu.be/dB11w>`_
```
Comment (bknueven) [inline README.rst:7]: Would it be better to use a direct link?

https://link.springer.com/article/10.1007/s12532-023-00247-3

File: README.rst [hunk 2]
```diff
@@ -51,19 +40,29 @@ an error message, mpi-sppy may still execute and return correct
...
+Installing mpi-sppy
+^^^^^^^^^^^^^^^^^^^
+
+It is possible to pip install mpi-sppy; however, most users are better off
+getting the software from github because it is under active development.
+
 Citing mpi-sppy
 ^^^^^^^^^^^^^^^
-If you find mpi-sppy useful in your work, we kindly request that you cite the following `pre-print <http://www.optimization-online.org/DB_HTML/2020/11/8088.html>`_:
+If you find mpi-sppy useful in your work, we kindly request that you cite the following `paper <https://rdcu.be/dB11w>`_:
```
Comment (bknueven) [inline README.rst:51]: Same here

**Review by bknueven — APPROVED** [review]
Just a hyperlink question.

File: README.rst (hunk 1 above)
Comment (bknueven) [inline README.rst:7]:
```suggestion
a there is a `paper <https://link.springer.com/article/10.1007/s12532-023-00247-3>`_
```

File: README.rst (hunk 2 above)
Comment (bknueven) [inline README.rst:51]:
```suggestion
If you find mpi-sppy useful in your work, we kindly request that you cite the following `paper <https://link.springer.com/article/10.1007/s12532-023-00247-3>`_:
```

**Pull request merged**